packages/lib/src/test/jsonld.test.ts

```diff
@@ -72,6 +76,17 @@ describe('check', () => {
     const result = await check(vc);
     expect(result.ok).toBeTruthy();
   });
+
+  it('should pass json with no mapped properties', async () => {
```

Comment:
would rather split this test since some checks don't match this text (some have mapped properties)